src/sentry/api/endpoints/project_releases.py

```diff
@@ -98,7 +98,7 @@ def get(self, request, project):
             queryset=queryset,
             order_by='-sort',
             paginator_cls=OffsetPaginator,
-            on_results=lambda x: serialize(x, request.user),
+            on_results=lambda x: serialize(x, request.user, project=project),
```

Comment:
Ah, that's an alternative approach. I didn't think of that. I suggested just letting the serializers accept `*args` and `**kwargs` to let the default serializer registry stuff work. Seemed the cleanest to me.